recipe/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "trulens-core" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trulens_core-{{ version }}.tar.gz
+  sha256: bfbadd53a5a5d17575a8855f3fd888cad47912eb53fc4b0f27487ad341ba9971
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 100
```

Comment:
Is there a reason to start with build number 100? If not, we should reset it to 0.

Reply:
It's a Snowflake specific requirement, It helps improve some workflows for them.